docs/source/operations/guide-troubleshooting.rst

```diff
@@ -25,6 +25,10 @@ When running Armada in the container you can execute docker logs to retrieve log
 
     docker logs [container-name | container-id]
 
+Errors/Exceptions
+-----------------
+
+A guide for interpreting errors/exceptions can be found `here <http://armada-helm.readthedocs.io/en/latest/operations/guide-api.html>`.
```

Comment:
also i belive you give it the path to the file and rtfd will actually put the correct link